concent_api/api-integration-force-get-task-result-test.py

```diff
@@ -32,6 +33,44 @@ def parse_command_line(command_line):
     return cluster_url
 
 
+def upload_new_file_on_cluster(task_id = '0', part_id = '0', current_time = 0):
+
+    os.system('touch result')
+    os.system('echo "{}" > result'.format(task_id))
+    file_size       = int(os.popen("wc -c < result").readlines()[0])
+    file_check_sum  = 'sha1:' + os.popen("sha1sum result | awk '{print $1}'").readlines()[0][:-1]
+    file_path       = '{}/{}/result'.format(task_id, part_id)
+
+    file_transfer_token = message.FileTransferToken()
+    file_transfer_token.token_expiration_deadline       = int(datetime.datetime.now().timestamp()) + 3600
+    file_transfer_token.storage_cluster_address         = STORAGE_CLUSTER_ADDRESS
+    file_transfer_token.authorized_client_public_key    = CONCENT_PUBLIC_KEY
+    file_transfer_token.operation                       = 'upload'
+
+    file_transfer_token.files                   = [message.FileTransferToken.FileInfo()]
+    file_transfer_token.files[0]['path']        = file_path
+    file_transfer_token.files[0]['checksum']    = file_check_sum
+    file_transfer_token.files[0]['size']        = file_size
+
+    upload_token    = shortcuts.dump(file_transfer_token, CONCENT_PRIVATE_KEY, CONCENT_PUBLIC_KEY)
+    encrypted_token = b64encode(upload_token).decode()
+
+    authorized_golem_transfer_token = 'Golem ' + encrypted_token
+
+    file = open('result', 'rb').read()
```

Comment:
You can attach file to request without physically creating it, please check 3rd example from
http://docs.python-requests.org/en/latest/user/quickstart/#post-a-multipart-encoded-file